shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -99,9 +99,11 @@ const mergeProps = (stateProps: StateProps, dispatchProps: DispatchProps, ownPro
     mobile: stateProps.mobile,
     muted: stateProps.muted,
     saveState: stateProps.saveState,
-    onMuteConversation: (muted: boolean) => {
-      dispatchProps._onMuteConversation(convKey, muted)
-    },
+    onMuteConversation: !Constants.isPendingConversationIDKey(convKey)
```

Comment:
moved from similar (now-unused) function from `info-panel/container`; not sure if it's really needed though